### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by SANYUE123321** [AlphaYuan/AutoUnmix#2]
Great work！Thanks a lot for open-source!
By using the dataset you provided, I found that I don't get the effect of the four-channel experimental data in your article. I checked your code and found that the loss function and the network structure don't match with what is shown in the article, is this where the problem appears? What should I do to solve this problem?
Thanks for ur reply!

**Comment by AlphaYuan** [comment]
Hello! 
Some loss functions are commented when debugging. You can uncomment corresponding lines. 
I have tested performance using current network structure which is simpler than that in the paper and it should produce similar results as in the paper.

**Comment by SANYUE123321** [comment]
Thank you for your reply!
